client/components/deposits-overview/index.tsx

```diff
@@ -57,7 +57,7 @@ const DepositsOverview: React.FC = () => {
 		! isLoading &&
 		deposits?.length > 0 &&
 		!! account &&
-		! account?.deposits_blocked;
+		isDepositsUnrestricted;
```

Comment:
I think we still want to show recent deposits if deposits are restricted. 
Actually recent deposits should show even if they're blocked IMO

Reply:
Good point, I'll test and remove this requirement.

Reply:
Updated in 867116c.

**Deposits hard blocked** – Deposits card not rendered

<img width="359" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/f8329a4a-8a6f-48be-bc5e-97b1356d2b38">

**Deposits soft blocked** – Deposits card not rendered

![image 2](https://github.com/Automattic/woocommerce-payments/assets/3147296/dbb1151f-95ce-414d-889a-cfc7e9fb6839)


**Deposits schedule restricted** – recent deposit list shown

![image 3](https://github.com/Automattic/woocommerce-payments/assets/3147296/fc429c82-c9f4-4c5b-ac51-cf455f6f6d7c)